Formula/ignition-transport9.rb

```diff
@@ -0,0 +1,59 @@
+class IgnitionTransport8 < Formula
+  desc "Transport middleware for robotics"
+  homepage "https://ignitionrobotics.org"
+  head "https://github.com/ignitionrobotics/ign-transport", :branch => "master"
```

Comment:
I have an updated version of this formula on https://github.com/osrf/homebrew-simulation/pull/1065, so that we can merge just this package and not be blocked by all others.